posts/2020-11-09-ocp-log-forwarding.adoc

```diff
@@ -305,7 +307,7 @@ Leave the other fields unchanged and click *Create a new index*. Copy the genera
 
 == Integrating Open Liberty JSON Logs
 
-Since Open Liberty application pods output logs in JSON format, it is recommended to set Fluentd to parse the JSON fields from the message body. To enable JSON parsing, change the cluster logging instance's *managementState* field from *"Managed"* to *"Unmanaged"*:
+Since Open Liberty application pods output logs in JSON format, it is recommended to set Fluentd to parse the JSON fields from the message body. To enable JSON parsing, first change the cluster logging instance's *managementState* field from *"Managed"* to *"Unmanaged"*:
```

Comment:
Since Open Liberty application pods output logs in JSON format, it is recommended to set Fluentd to parse the JSON fields from the message body. To enable JSON parsing, first change the cluster logging instance's *managementState* field from *"Managed"* to *"Unmanaged"*:

--->

Open Liberty application pods output logs in JSON format. To enable Fluentd to parse the JSON fields from the message body, first change the cluster logging instance's *managementState* field from *"Managed"* to *"Unmanaged"*:

Reply:
fixed